x-pack/plugins/secops/public/components/timeline/body/index.tsx

```diff
@@ -21,6 +21,7 @@ interface Props {
   columnHeaders: ColumnHeader[];
   data: ECS[];
   dataProviders: DataProvider[];
+  height?: number | string;
```

Comment:
Correct me if I'm wrong but I don't think you can't use a number at all right at the moment since you have this below:

```
style={{ width: `${width - 10}px`, height }}>
```
You would end up with:
<img width="134" alt="screen shot 2018-11-06 at 8 42 23 pm" src="https://user-images.githubusercontent.com/1151048/48109403-90176900-e204-11e8-8363-f94bbc7173d0.png">

You probably only want that a string. To make things more flexible and consistent we probably should make width just a string as well so we can pass down 100% or 100px, etc... I would just remove the number type for now and keep this a string and we can refactor the width next and decide if both width and height should be optional with defaults or not and other things. Keep width and height as two peas in a pod.